src/rpm.rs

```diff
@@ -219,6 +288,13 @@ impl PartialEq for Lead {
     }
 }
 
+pub trait Tag {}
```

Comment:
Is this actually used?

Reply:
Not yet, it's a simplifying trait bound so one does not have to type the individual trait collections.